src/main/java/sirius/biz/jupiter/JupiterConnector.java

```diff
@@ -25,16 +25,28 @@
 /**
  * Permits to access a <b>Jupiter</b> instance.
  * <p>
- * This connector is in charge of taking care about the connection management. It also permits to failover to
+ * This connector is in charge of taking of about the connection management. It also permits to failover to
  * a fallback instance in case the main instance isn't reachable.
  * <p>
- * If a failover is performed, we will continute to use the fallback for up to 60s before we attempt to switch
+ * If a failover is performed, we will continue to use the fallback for up to 60s before we attempt to switch
  * back to the main instance.
  */
 public class JupiterConnector {
 
     private static final JupiterCommand CMD_SET_CONFIG = new JupiterCommand("SYS.SET_CONFIG");
 
+    /**
+     * If a failover is performed, we keep using the fallback instance for a certain amount of time to prevent
+     * constant switching back and forth in case of network problems (etc). This constant determines the interval
+     * before a failback is attempted.
```

Comment:
```suggestion
     * before a failover is attempted.
```

Reply:
failback is the attempt to shorten "a failover back to the main instance"

Reply:
thanks for the clarification!